README.md

```diff
@@ -1,8 +1,4 @@
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/magnars/dash.el/CI)
-[![MELPA](https://melpa.org/packages/dash-badge.svg)](https://melpa.org/#/dash)
-[![MELPA Stable](https://stable.melpa.org/packages/dash-badge.svg)](https://stable.melpa.org/#/dash)
-
-# <img align="right" src="https://raw.github.com/magnars/dash.el/master/rainbow-dash.png"> dash.el
+# <img align="right" src="https://raw.github.com/magnars/dash.el/master/rainbow-dash.png"> dash.el ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/magnars/dash.el/CI)
```

Comment:
Could you please re-run `create-docs.sh`? I'd forgotten about `readme-template.md` when I merged the last PR that added these badges. Some unrelated changes below should also be covered by that.

Reply:
I've update documentation and examples, and tried to run `create-docs.sh`. It worked alright, except for `dash.info`. Most of the file seems to be altered and changes feel unnecessary, like using double spaces after periods and using
`’function’` instead of ``` `function' ```
I've tried updating makeinfo, checked PATH in case it points to an older version,  but I don't know what to do.

Reply:
You could just leave out the changes to the manual from this PR and I or someone else will update them after the fact.

[I'd like to move away from checking in the generated Info manual. I don't know why it was added to begin with, but if it was related to GNU ELPA's tooling then that should have been recently fixed.]

Reply:
Very well! I'll focus on the code and examples. 😄